TestBenches/MatchEngine_test.cpp

```diff
@@ -27,9 +28,16 @@ using namespace std;
   #define KLAYERDISK 2 // Corresponds to TF::L3
 #endif
 constexpr auto kLayerDisk = static_cast<TF::layerDisk>(KLAYERDISK);
+#if !defined KMODULE
+	#define KMODULE ME_L3PHIC20_
+#endif
 #if !defined TOPFUNCTION
   #define TOPFUNCTION MatchEngineTop_L3
 #endif
+template<TF::layerDisk LayerDisk> constexpr int getLayerDiskNumber() {
+	// Convert from the enum index (0-based) to the module name indexing (1-based)
+  return LayerDisk <= TF::L6 ? LayerDisk + 1 : LayerDisk - 5;
```

Comment:
@aperloff This is super nitpicky, but could you replace `5` with `trklet::N_LAYER - 1`? The constant is defined, so we might as well use it.

Reply:
Done